src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Services/SftpService.cs

```diff
@@ -8,6 +8,7 @@ public class SftpService : ISftpService
 {
     private readonly ILogger<SftpService> _logger;
     private readonly SftpClient _client;
+    private bool _disposed;
```

Comment:
Should we fix this warning?

Reply:
Thanks for catching that. I think it would be fine to fix in develop after merge